libs/backend/src/cast_vote_records/export.ts

```diff
@@ -359,6 +372,33 @@ async function exportCastVoteRecordFilesToUsbDrive(
       canonicalizedSheet.interpretation.some(({ votes }) => hasWriteIns(votes))
     : true;
 
+  const castVoteRecordFilesToExport: ReadableFile[] = [];
```

Comment:
Had to change the order of operations a bit. I now export images and layout files first (if applicable) and then the `cast-vote-record-report.json`, as opposed to the other way around, since the `cast-vote-record-report.json` needs the hashes of the images and layout files